tools/Google.Cloud.Tools.VersionCompat/Google.Cloud.Tools.VersionCompat.csproj

```diff
@@ -8,7 +8,7 @@
   <ItemGroup>
     <PackageReference Include="Mono.Cecil" Version="0.10.3" />
     <PackageReference Include="SharpCompress" Version="0.24.0" />
-    <PackageReference Include="System.Collections.Immutable" Version="1.7.0" />
+    <PackageReference Include="System.Collections.Immutable" Version="1.7.1" />
```

Comment:
I'm not sure why this update is here, I will revert it here and let renovate create a new PR for it on its own.

Reply:
```suggestion
    <PackageReference Include="System.Collections.Immutable" Version="1.7.0" />
```